### PR TITLE
Migrate tests from TL2.fund() to FTLA.fund() in TrueFiPool2.test.ts

### PR DESCRIPTION
**Pull request by skalermo**

### CODE REVIEW

**Comment by skalermo** [comment]
I'm aware that SAFU tests are failing, those are handled in #1010. Once #1010 is merged this PR should be ready to go.

File: contracts/truefi2/FixedTermLoanAgency.sol
Comment (vanruch) [inline contracts/truefi2/FixedTermLoanAgency.sol:231]: Can we avoid adding unnecessary setters to contracts?

Comment (yuchenlintt) [reply contracts/truefi2/FixedTermLoanAgency.sol:231]: +1.

If we're launching the first version with TrueRateAdjuster (=> CreditModel) in the initializer, then we won't need to add a setter.

File: test/truefi2/TrueFiPool2.test.ts
Comment (yuchenlintt) [inline test/truefi2/TrueFiPool2.test.ts:746]: Ah, I see what setRateAdjuster was for.

For tests like this, would it work to pass this mockRateAdjuster in to setupTrueFi2 as an argument?

**Review by yuchenlintt — CHANGES_REQUESTED** [review]
I'll merge 1010 in, and then let's see if we can avoid adding a `setRateAdjuster()` function.

**Review by yuchenlintt — APPROVED** [review]

File: test/truefi2/SAFU.test.ts
Comment (MiksuJak) [inline test/truefi2/SAFU.test.ts:13]: image

**Review by vanruch — APPROVED** [review]

**Pull request merged**
